collector/filesystem_common.go

```diff
@@ -80,6 +81,7 @@ type filesystemStats struct {
 	labels            filesystemLabels
 	size, free, avail float64
 	files, filesFree  float64
+	purgeable         float64
```

Comment:
This adds purgeable to all platforms. I would recommend making this a `*float64` and only returning the metric like this:

```
if s.purgeable != nil {
  ch <- prometheus.MustNewConstMetric(...)
```